R/AddModule.R

```diff
@@ -0,0 +1,15 @@
+#' @name AddModule
```

Comment:
Now that we have AddTab should the default for AddModule and AddModules have the  bDashboard = FALSE ?